composer.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "silverstripe/silverstripe-session-manager",
+	"name": "silverstripe/session-manager",
```

Comment:
We already published to Packagist as `silverstripe/silverstripe-session-manager`, but I suppose renaming it prior to any formal release is the only opportunity we'll have.